### PR TITLE
Created a rough draft of Profile Page

### PR DESCRIPTION
**Pull request by nkonjeti**

### CODE REVIEW

**Comment by coconut750750** [comment]
Looks pretty good

File: frontend/src/pages/Profile.js
Comment (alicesf2) [inline frontend/src/pages/Profile.js]: I would suggest changing the variable name to something more specific, like profile

File: frontend/src/styles/Profile.css
Comment (alicesf2) [inline frontend/src/styles/Profile.css:1]: good use of css classes :D

File: backend/index.js
Comment (alicesf2) [inline backend/index.js]: nitpick: try to camelcase all javascript variables (i.e. userid -> userId)

File: frontend/src/utils/api.js
Comment (alicesf2) [inline frontend/src/utils/api.js]: change this to userId too

**Review by alicesf2 — COMMENTED** [review]
Looks good :nehalapeno:! I'll wait until you finish the edit profile page to approve

File: backend/index.js
Comment (coconut750750) [inline backend/index.js]: get rid of some of these comments

**Review by coconut750750 — CHANGES_REQUESTED** [review]

File: backend/index.js
Comment (nkonjeti) [reply backend/index.js]: 👍🏽

**Review by coconut750750 — APPROVED** [review]

**Pull request merged**
